packages/amplify_core/lib/src/logger/simple_printer.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:logging/logging.dart';
+
+class SimplePrinter implements AmplifyLoggerPlugin {
+  SimplePrinter();
+
+  @override
+  void handleLogRecord(LogRecord record) {
+    final buffer = StringBuffer();
+
+    // Log Level
+    buffer.write(record.level.toString());
```

Comment:
Do we want a separator after the level and/or namespace? Probably wasn't needed with the pretty print. With the simple print, it might be nice to have a separator. It all runs together as is.

Current
```
INFO namespace message
INFO namespace message
WARN namespace message
```

Some other options
```
INFO: namespace | message
INFO: namespace | message
WARN: namespace | message
```

```
INFO | namespace | message
INFO | namespace | message
WARN | namespace | message
```

Reply:
Sounds good, added some very basic formatting: 

![image](https://user-images.githubusercontent.com/11478656/178775302-c9622af2-8f2a-400d-8b5e-1291bc267bb4.png)